.github/workflows/podman-desktop-e2e-stress-ui-linux.yaml

```diff
@@ -0,0 +1,177 @@
+name: Podman Desktop E2E UI Stress Test (Linux)
+run-name: Podman Desktop E2E UI Stress Test (Linux)
+
+on:
+  schedule:
+    - cron: '0 5 * * 1' #run every monday at 5AM
+  workflow_dispatch:
+    inputs:
+      pd_repo_options:
+        default: 'REPO=podman-desktop,FORK=podman-desktop,BRANCH=main'
+        description: 'Podman Desktop repo, fork and branch'
+        type: string
+        required: true
+      npm_target:
+        default: 'test:e2e:ui-stress'
+        description: 'npm target to run tests'
+        type: string
+        required: true
+      podman_remote_url:
+        default: 'https://github.com/containers/podman/releases/download/v5.4.0/podman-remote-static-linux_amd64.tar.gz'
+        description: 'podman latest released version exe'
+        type: string
+        required: true
+      podman_options:
+        default: 'INIT=1,START=1,ROOTFUL=0,NETWORKING=0'
+        description: 'Podman machine configuration options, no spaces'
+        type: 'string'
+        required: true
+      env_vars:
+        default: 'ELECTRON_ENABLE_INSPECT=true,OBJECT_NUM=100'
```

Comment:
if you want to paramterize the OBJECT_NUM, you should set it as individual input. inside env_vars input string it does not work like that.